netmaster/master/endpointGroup.go

```diff
@@ -149,6 +150,10 @@ func DeleteEndpointGroup(tenantName, groupName string) error {
 		return err
 	}
 
+	if epgCfg.EpCount != 0 {
+		return core.Errorf("Error: EPG has active endpoints")
```

Comment:
can we mention the epg name here ?  So while debugging we can use epg inspect to list the active endpoints.